bonetrousle.py

```diff
@@ -0,0 +1,52 @@
+#!/bin/python3
+
+import os
+import sys
+
+#
+# Complete the bonetrousle function below.
+#
+
+def bonetrousle(n, k, b):
+    #
+    # Write your code here.
+    min_sum = b*(b+1)//2
+    max_sum = b*((k-b+1)+k)//2 

```

Comment:
min_sum/max_sum need an explanation as it's really unclear what you're trying to do and whether it's even legit.

Also a comment about the algorithm you're using if it's well-known would help as I don't recognize it.